src/racing/memory.c

```diff
@@ -938,7 +939,7 @@ void unpack_spline_3D(Gfx *gfx, u8 *arg1, UNUSED s8 arg2) {
         temp_v0 = arg1[sPackedSeekPosition++];
         phi_a0 |= (temp_v0 & 0xF) * 2;
     }
-    gfx[sGfxSeekPosition].words.w0 = (G_LINE3D << 24);
+    gfx[sGfxSeekPosition].words.w0 = ((uintptr_t)(uint8_t)G_LINE3D << 24);
     gfx[sGfxSeekPosition].words.w1 =
```

Comment:
These two lilnes should probably say G_QUAD not G_LINE3D as line3d only exists in the line3d microcode and not in f3dex.